packages/strapi-plugin-graphql/services/Mutation.js

```diff
@@ -155,7 +159,7 @@ module.exports = {
     }
 
     if (strapi.plugins['users-permissions']) {
-      policies.push('plugins.users-permissions.permissions');
+      policies.unshift('plugins.users-permissions.permissions');
```

Comment:
`unshift` will put the `permissions` policy before the `global` policy which can lead to errors, because `permissions` depends on `ctx.request.route` which is added in the `global` policy.